docusaurus/docs/how-to-guides/ui-extensions/register-an-extension.md

```diff
@@ -0,0 +1,234 @@
+---
+id: register-an-extension
+title: Register an extension
+description: Make part of the UI extendable with links or components defined by plugins
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - extensions
+  - ui-extensions
+sidebar_position: 20
+---
+
+import ExtensionPoints from '@shared/extension-points.md';
+
+An _extension_ is a plugin-defined link or a React component that is rendered in either the core Grafana UI or in another app plugin.
```

Comment:
```suggestion
An _extension_ is a plugin-defined link or a React component that is rendered in either the core Grafana UI or in another app plugin.

Compared to [exposing a component](./expose-a-component.md), they are explicitly registered against one or more extension point IDs. This can be more appropriate when looking to extend Grafana's core UI, or for when you need more control over what should be allowed to use your plugin's extension.
```

Reply:
Felt we needed something here, please check to see if you agree with the statement.